x-pack/test/performance/tests/home.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['common', 'security']);
+  const testSubjects = getService('testSubjects');
+  const retry = getService('retry');
+
+  describe('Login', () => {
+    it('login and navigate to homepage', async () => {
+      await PageObjects.common.navigateToApp('login');
```

Comment:
For these tests, I think we might want to avoid these helpers or maybe improve them? There is some crazy retry and sleeps in [here](https://github.com/elastic/kibana/blob/main/test/functional/page_objects/common_page.ts#L221-L298). I

Reply:
I agree with you that there are quite extensive usage of retry logic one in another. However I also felt like since this `PageObjects.common.navigateToApp` is pretty generic if we remove/change the retries and waits in that function there can be some tests starts to fail because of timeouts. I spent some time on weekend to understand what is the best way of verifying (waiting explicitly, implicitly or retries) to land a page or check existence of an element, what I found out is there is no silver bullet for solving these issues with some magical logic each use case needs to be evaluated separately. In our case I believe navigateToApp function is trying to be as defensive as possible by adding multiple retries inside of each other. I think it would be nice if @spalger also adds some comments on this before I change something.

Reply:
Can we log a test user in asynchronously to minimize the number of side effects affecting the result of performance testing?

Reply:
Yes for sure we can try it out 👍🏼. However I'm not familiar how to add test user 😅 and what would be the difference between this login flow and with the test user in terms of side effects

Reply:
My opinion is that we should consider alternate ways to navigate to the page only if we determine that it's a problem when we're trying to get more stable metrics and we've determined from the data that the way we're navigating is causing issues. Navigation is complex, making an implementation that works just for the performance stuff shouldn't be too bad, but "fixing" the common page object is a bit of a minefield if you ask me. There are a lot of considerations... Just my opinion.

Reply:
I am fine moving forward with this while we generate data on the deviation. At this point, we're still not certain if we will be using the FTR, or will need bare metal.